Firebase/Auth/Source/Auth/FIRAuth.m

```diff
@@ -384,6 +385,11 @@ - (nullable instancetype)initWithAPIKey:(NSString *)APIKey appName:(NSString *)a
     UIApplication *application = [applicationClass sharedApplication];
 
     [GULAppDelegateSwizzler proxyOriginalDelegateIncludingAPNSMethods];
+    #if ((TARGET_OS_IOS || TARGET_OS_TV) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= 130000))
```

Comment:
Using Macro does not work in framework as the this will be built before developers can use anyway. Please use runtime check only

Reply:
Both compile-time and run-time checks are needed. Most developers use Auth as a source-based CocoaPod.